### ISSUE TITLE
Please provide the instruction for deploying on keycloak server with domain setup

### ISSUE THREAD

**Issue by kappa-wingman** [privacyidea/keycloak-provider#69]
Keycloak server with domain server (not standalone) do not auto deploy any jar that was put in the deployments folder. Auto deploy only works for standalone keycloak setup.

So please provide the instruction for deployment.
I tried to use the instruction in https://www.keycloak.org/docs/latest/server_development/#register-a-provider-using-modules but failed to deploy the keycloak provider.

**Comment by adamboutcher** [comment]
+1 for some instructions. Keycloak 18 doesnt seem to load this...

**Comment by adamboutcher** [comment]
For Keycloak 18 just pop the jar file in providers and rebuild.

**Comment by nilsbehlen** [comment]
yes it works for keycloak 18. However, for "domain server" setup i do not know and i dont have the time to learn how to set it up.

**Comment by adamboutcher** [comment]
I suspect the domain setup is the same as standalone as keycloak docs don't differentiate it as much in the new deployment.
